tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0036_course_enrollments.py

```diff
@@ -0,0 +1,39 @@
+from alembic import op
+import sqlalchemy as sa
+
+revision = "0036"
+down_revision = "0035"
+branch_labels = None
+depends_on = None
+on_cluster = " ON CLUSTER '{{CLICKHOUSE_CLUSTER_NAME}}' " if "{{CLICKHOUSE_CLUSTER_NAME}}" else ""
+engine = "ReplicatedMergeTree" if "{{CLICKHOUSE_CLUSTER_NAME}}" else "MergeTree"
+
+
+def upgrade():
+    op.execute(
+        f"""
+        -- Raw table that Ralph writes to
+        CREATE TABLE IF NOT EXISTS {{ ASPECTS_EVENT_SINK_DATABASE }}.course_enrollment
+        {on_cluster}
+        (
+            id Int32 NOT NULL,
+            course_key String NOT NULL,
+            created String NOT NULL,
```

Comment:
This should probably be a DateTime, right?

Reply:
In theory, however we adjust this on the serializer to make it compatible with what clickhouse expects, this will not be used directly like the other sinks, a MV will be created to query this table, so let me know what do you think I should do for it. Other sinks also uses strings